database.sql

```diff
@@ -1,2 +1,26 @@
-CREATE DATABASE IF NOT EXISTS groupc_kainos_recruitment;
-USE groupc_kainos_recruitment;
+CREATE DATABASE IF NOT EXISTS groupc_kainos_recruitment_JenniferC;
+USE groupc_kainos_recruitment_JenniferC;
+
+CREATE TABLE IF NOT EXISTS job_roles (
+id smallint PRIMARY KEY AUTO_INCREMENT,
+name varchar(50),
+job_description varchar(500)
+);
+
+-- Engineering Capability
+INSERT INTO job_roles (name, job_description) VALUES ('Software Engineer', 'You will work on projects where you can make a real difference to people’s lives – the lives of people you know. Undertake extensive training to set you off on the right foot, you will quickly work as a part of a team in developing solutions within our real projects and learn all about our development languages, projects and technologies. You will be fully supported by experienced colleagues in the team, as well as an experienced mentor, who will provide training and mentoring throughout your studies.');
```

Comment:
One sentence for each would really do, but happy if you want to leave it chunky like this

Reply:
We're happy enough with leaving the insert statements as is